Maddori.Apple/Maddori.Apple/Screen/MyReflection/MyReflectionFeedback/MyReflectionFeedbackViewController.swift

```diff
@@ -87,56 +85,58 @@ final class MyReflectionFeedbackViewController: BaseViewController {
+            $0.top.equalTo(view.safeAreaLayoutGuide).inset(SizeLiteral.topPadding)
             $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
         }
         
-        myReflectionContentView.addSubview(feedbackTypeLabel)
+        view.addSubview(feedbackTypeLabel)
         feedbackTypeLabel.snp.makeConstraints {
             $0.top.equalTo(keywordTitleLabel.snp.bottom).offset(SizeLiteral.topComponentPadding)
             $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
         }
         
-        myReflectionContentView.addSubview(feedbackTypeText)
+        view.addSubview(feedbackTypeText)
         feedbackTypeText.snp.makeConstraints {
-            $0.top.equalTo(feedbackTypeLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
-            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.centerY.equalTo(feedbackTypeLabel.snp.centerY)
+            $0.leading.equalTo(feedbackTypeLabel.snp.trailing).offset(54)
         }
         
-        myReflectionContentView.addSubview(feedbackFromLabel)
+        view.addSubview(feedbackFromLabel)
         feedbackFromLabel.snp.makeConstraints {
-            $0.top.equalTo(feedbackTypeText.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.top.equalTo(feedbackTypeText.snp.bottom).offset(20)
             $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
         }
         
-        myReflectionContentView.addSubview(feedbackFromText)
+        view.addSubview(feedbackFromText)
         feedbackFromText.snp.makeConstraints {
-            $0.top.equalTo(feedbackFromLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
-            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.centerY.equalTo(feedbackFromLabel.snp.centerY)
+            $0.leading.equalTo(feedbackFromLabel.snp.trailing).offset(86)
         }
         
-        myReflectionContentView.addSubview(feedbackContentLabel)
-        feedbackContentLabel.snp.makeConstraints {
-            $0.top.equalTo(feedbackFromText.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
-            $0.leading.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        view.addSubview(divider)
+        divider.snp.makeConstraints {
+            $0.top.equalTo(feedbackFromLabel.snp.bottom).offset(24)
+            $0.horizontalEdges.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.height.equalTo(1)
         }
         
+        view.addSubview(myReflectionScrollView)
+        myReflectionScrollView.snp.makeConstraints {
+            $0.top.equalTo(divider.snp.bottom)
+            $0.horizontalEdges.bottom.equalTo(view.safeAreaLayoutGuide)
+        }
+
+        myReflectionScrollView.addSubview(myReflectionContentView)
+        myReflectionContentView.snp.makeConstraints {
+            $0.width.top.bottom.equalToSuperview()
```

Comment:
p2;

<img width="740" alt="image" src="https://user-images.githubusercontent.com/78677571/209918087-3bc9eecd-ae57-4069-8d6c-cba34465f658.png">
현재 이 부분에서 레이아웃 경고가 뜨네요 !


```suggestion
            $0.edges.width.equalToSuperview()
```

전체적인 레이아웃과 width를 같이 잡아주면 해결할 수 있습니다 !

Reply:
[7be13d8](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/239/commits/7be13d8bf7b0accddf817a420f2ffb2fa4883446)
와우 섬세한 리뷰 감사합니당 ㅎㅎ 수정해서 반영했습니다!